src/main/kotlin/org/jetbrains/bio/npy/DictParser.kt

```diff
@@ -41,7 +41,7 @@ private fun parseTuple(lexer: PeekingIterator<SpannedToken>): List<Int> {
     val acc = ArrayList<Int>()
     while (lexer.peek().token != Token.RPAR) {
         val item = lexer.eat(Token.INT)
-        acc.add(item.span.toInt())
+        acc.add(item.span.filterNot { c -> c == 'L' }.toInt())
```

Comment:
Could you change this to `substringBeforeLast('L')`? It is operationally identical to your version, but illustrates the fact that we only expect L in the end.